proto/poktroll/supplier/tx.proto

```diff
@@ -9,33 +10,32 @@ import "cosmos/msg/v1/msg.proto";
 import "cosmos_proto/cosmos.proto";
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
-
 import "poktroll/supplier/params.proto";
 import "poktroll/shared/service.proto";
 
-
 // Msg defines the Msg service.
 service Msg {
   option (cosmos.msg.v1.service) = true;
-
+  
   // UpdateParams defines a (governance) operation for updating the module
   // parameters. The authority defaults to the x/gov module account.
   rpc UpdateParams    (MsgUpdateParams   ) returns (MsgUpdateParamsResponse   );
   rpc StakeSupplier   (MsgStakeSupplier  ) returns (MsgStakeSupplierResponse  );
   rpc UnstakeSupplier (MsgUnstakeSupplier) returns (MsgUnstakeSupplierResponse);
+  rpc UpdateParam     (MsgUpdateParam    ) returns (MsgUpdateParamResponse    );
 }
 // MsgUpdateParams is the Msg/UpdateParams request type.
 message MsgUpdateParams {
-  option (cosmos.msg.v1.signer) = "authority";
-  option (amino.name) = "poktroll/x/supplier/MsgUpdateParams";
-
+  option (cosmos.msg.v1.signer) =                           "authority";
+  option           (amino.name) = "poktroll/x/supplier/MsgUpdateParams";
+  
```

Comment:
```suggestion

```

Reply:
My personal preference is to have the `option`s separate from fields by one blank line. I have not surveyed the proto messages to see what's more consistent nor am I aware of any particular convention about this. Do you have a strong opinion or a rationale you could chare?